packages/domains-analytics-writer/src/model/attribute/attribute.ts

```diff
@@ -0,0 +1,23 @@
+// attribute.ts
+import { AttributeSQL } from "pagopa-interop-readmodel-models";
+import { z } from "zod";
+
+export const attributeSchema = z.object({
```

Comment:
Even though you don't use Drizzle to connect to the DB and make queries, I think you can still use Drizzle just to generate these models from the schemas you define, if useful, to keep them in sync.
(If these schemas are all meant to be super simple and not to change over time, maybe it's not so useful.)